packages/edit-site/src/components/global-styles/screen-revisions/index.js

```diff
@@ -153,70 +150,81 @@ function ScreenRevisions() {
 				) }
 				onBack={ onCloseRevisions }
 			/>
-			{ isLoading && (
+			{ ! hasRevisions && (
 				<Spinner className="edit-site-global-styles-screen-revisions__loading" />
 			) }
-			{ shouldShowRevisions && (
-				<>
+			<>
+				{ hasRevisions && (
 					<Revisions
 						blocks={ blocks }
 						userConfig={ currentlySelectedRevision }
 						onClose={ onCloseRevisions }
 					/>
-					<div className="edit-site-global-styles-screen-revisions">
-						<RevisionsButtons
-							onChange={ selectRevision }
-							selectedRevisionId={ currentlySelectedRevisionId }
-							userRevisions={ revisions }
-							canApplyRevision={ isLoadButtonEnabled }
-						/>
+				) }
+				<div className="edit-site-global-styles-screen-revisions">
+					<RevisionsButtons
+						onChange={ selectRevision }
+						selectedRevisionId={ currentlySelectedRevisionId }
+						userRevisions={ currentRevisions }
+						canApplyRevision={ isLoadButtonEnabled }
+					/>
+				</div>
+				{ hasRevisions && (
+					<div className="edit-site-global-styles-screen-revisions__footer">
 						{ isLoadButtonEnabled && (
-							<SidebarFixedBottom>
```

Comment:
Follow up for this PR:

We might be able roll back `<SidebarFixedBottom>` completely as it was implemented for this panel only in https://github.com/WordPress/gutenberg/pull/50089/files#diff-5455ae916aab538689ffe932f42050ae3bcb29be2213af3f25f65abf3ec40b14R11.

Reply:
No, we need it now as a sticky object will jump depending on the height of the results container.